Podfile.lock

```diff
@@ -154,7 +154,7 @@ SPEC CHECKSUMS:
   HKDFKit: c058305d6f64b84f28c50bd7aa89574625bcb62a
   JSQMessagesViewController: 39fed975e3c9f8eba7292071e29eeb541d105e66
   JSQSystemSoundPlayer: c5850e77a4363ffd374cd851154b9af93264ed8d
-  libPhoneNumber-iOS: 81ad1e6bfcf46e668636333269afbfe60399a55b
+  libPhoneNumber-iOS: a8bffdec18c37728360f6771fe021302f1e0b497
```

Comment:
Pod update updates all the dependencies below SSK too. 

Could you update libphonenumber-ios separately for future git bisectabilitiy?

I would do this by manually editing the podfile.lock (or `git co -p master -- Podfile.lock`!), then re-running `pod install` to get a sensible PODFILE CHECKSUM.